app/views/users/index.html.erb

```diff
@@ -10,32 +10,26 @@
   <h1 class="mt-5">All Users</h1>
   <p class="lead">Here you can see all the users of the blog.</p>
   <div class="row">
-    <ul class="list-group mt-4 w-100">
+    <ul class="list-group my-4 w-100">
       <% @users.each do |user| %>
-        <li class="list-group-item">
-          <div class="row">
-            <div class="col-sm-3">
-              <div class="d-flex align-items-center">
-                <div class="user-photo mr-3">
-                  <% if user.photo && user.photo.match(/\.(png|jpg|jpeg|gif)$/i) %>
-                    <img src="<%= user.photo %>" alt="User Photo" style="width: 150px">
-                  <% else %>
-                    PhotoUSer<%= user.id %>
-                  <% end %>
+        <li class="list-group-item my-2 border">
+          <div class="media align-items-center">
+            <div class="user-photo mr-3">
+              <% if user.photo && user.photo.match(/\.(png|jpg|jpeg|gif)$/i) %>
+                <img src="<%= user.photo %>" alt="User Photo" style="width: 150px; height: 150px;">
+              <% else %>
+                <div class="placeholder-image mr-3" style="width: 150px; height: 150px;">
+                  PhotoUser<%= user.id %>
                 </div>
-              </div>
+              <% end %>
```

Comment:
Your design
![image](https://github.com/sergio-usma/rails_blog_pj/assets/78251927/557e973e-3c4e-460a-860d-9eb79993d301)

wireframe
![image](https://github.com/sergio-usma/rails_blog_pj/assets/78251927/5f63e546-edb7-4cff-b252-e1b340da9290)

To match the provided wireframe, kindly add a gap between the name and the photo

Reply:
Done!